spec/features/park_description_tests/01_scrapes_description_from_NPS_park_page_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+feature "ParkDescription scans the NPS.gov page for each park to update it's brief description" do
+  scenario "method pulls the park homepage description from the park.nps_page to display on park show page" do
+    user = FactoryGirl.create(:user)
+    park_01 = FactoryGirl.create(:park,
+      user_id: user.id,
+      nps_page: "https://www.nps.gov/arch/index.htm",
+      main_image: Rack::Test::UploadedFile.new(File.open(File.join(Rails.root, '/spec/support/mountains_01.jpg'))))
```

Comment:
Since you aren't testing that this image exists you probably don't need or want to upload this image just for testing. If you have a wrapper css for images, testing for that's existence will be faster and easier to test for on a page.